# Multiple tracked objects with motion information
std_msgs/Header header
TrackedObject[] objects  # Array of tracked objects
int32 total_tracks      # Total number of active tracks